### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by huzaifakhan0308**
I make this project using react-redux.

### Highlights

- Home page
- Detail page
- APIS
- react-router
- redux-thunk
- styles
- write tests with a React testing library

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,100 @@
...
+- **[login & sign up]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Feel free to check [issues page](https://github.com/huzaifakhan0308/react-capston/issues).
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
```
Comment (Mihndim2020) [inline README.md:93]: - [ ] As it is a [requirement ](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md)for this project, "`The Creative Commons license of the design requires that you give appropriate credit to the author. Therefore, you must do it in the README of your project.
`", please kindly acknowledge [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio) for his original design idea.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @huzaifakhan0308, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Clean and Organized code
:heavy_check_mark: Good job with UI
:heavy_check_mark:  Good with video presentations
:heavy_check_mark: Good job with test coverage 
:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1 +1,102 @@
...
+- More details about the project can be seen in this [video](https://www.loom.com/share/2d54f1e084ca4ed388b09794ed6a21a7) 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [live link](https://weather-tcql.onrender.com)
+
+- The design was inspired [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio)
+
```
Comment (elisha2kyakpo1) [inline README.md]: - [ ] Kindly add this line under the `Acknowledgments` section, this will make the readme to look organized and professional.

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @huzaifakhan0308, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

👍 You did well to make the changes requested by the previous reviewer.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @huzaifakhan0308, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**